### DISCUSSION TITLE
Linux issue

### DISCUSSION THREAD

**Discussion by ASB224** [community/community#36201] [Actions]
Hello Mr Tek I am having some issue with subscription-manager register 
After I enter my iD and passwd it’s not responding and it show a file like /var/ something I don’t really remember and your WhatsApp and telegrams links have expired please help thanks

**Comment by ossanna16** [comment]
Hi @ASB224,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Linux. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
